tests/test_recipient_validation.py

```diff
@@ -40,55 +40,32 @@
 valid_phone_numbers = valid_local_phone_numbers + valid_international_phone_numbers
 
 
-invalid_local_phone_numbers = sum(
-    [
-        [(phone_number, error) for phone_number in group]
-        for error, group in [
-            (
-                "Not a valid local number",
-                (
-                    "712345678910",
-                    "0712345678910",
-                    "0044712345678910",
-                    "0044712345678910",
-                    "+44 (0)7123 456 789 10",
-                ),
-            ),
-            (
-                "Not a valid local number",
-                (
-                    "0712345678",
-                    "004471234567",
-                    "00447123456",
-                    "+44 (0)7123 456 78",
-                ),
-            ),
-            (
-                "Not a valid local number",
-                (
-                    "08081 570364",
-                    "+44 8081 570364",
-                    "0117 496 0860",
-                    "+44 117 496 0860",
-                    "020 7946 0991",
-                    "+44 20 7946 0991",
-                ),
-            ),
-            (
-                "Not a valid local number",
-                (
-                    "07890x32109",
-                    "07123 456789...",
-                    "07123 ☟☜⬇⬆☞☝",
-                    "07123☟☜⬇⬆☞☝",
-                    "+44 07ab cde fgh",
-                    "ALPHANUM3R1C",
-                ),
-            ),
-        ]
-    ],
-    [],
-)
+invalid_local_phone_numbers = [
```

Comment:
I executed this statement and found that it was just a really over-complicated way of building a list like this:
```
[("1234", "Not a valid local number"), ("2345", "Not a valid local number"), ...]
```

So I replaced it, verified that the new statement `==` the old, and the type error went away.

Reply:
Thanks for coming back on this; the code is much clearer to me with this new version.